util/resolutionizer.py

```diff
@@ -304,6 +304,73 @@ def miller_array_from_mtz(unmerged_mtz, params):
 )
 
 
+def cc_half_fit(
+    merging_statistics,
+    significance_level=0.1,
+    cc_half_method="half_dataset",
+    cc_half_fit="tanh",
+    limit=0.3,
+):
+
+    if cc_half_method == "sigma_tau":
+        cc_s = flex.double(
+            [b.cc_one_half_sigma_tau for b in merging_statistics.bins]
```

Comment:
```suggestion
            b.cc_one_half_sigma_tau for b in merging_statistics.bins
```
flex constructors work with generators, no need to explicitly construct a list and then throw it away. Applies to this line and a couple more times down the file.